net/freeradius/src/opnsense/mvc/app/controllers/OPNsense/Freeradius/forms/eap.xml

```diff
@@ -0,0 +1,32 @@
+<form>
+    <field>
+        <id>eap.default_eap_type</id>
+        <label>Default EAP Type</label>
+        <type>dropdown</type>
+        <help>Set the default EAP type, in most situations MD5 is good.</help>
```

Comment:
This working is really bad, because "MD5" != "good". Maybe simply remove the second part of the sentence?

Reply:
md5 is most compatible for most screnarios, that's why FR put it as default. I'll remove the second sentence. The meaning was to only change it if the person really knows what to do :)

Reply:
understood. MD5 in the form as a default is enough. more may or may not be mentioned in the docs in terms of compatibility, not "goodness" :)